src/commands/Suggestions/suggest.ts

```diff
@@ -0,0 +1,99 @@
+import { CommandStore, KlasaMessage } from 'klasa';
+import { MessageEmbed, TextChannel } from 'discord.js';
+import { SkyraCommand } from '@lib/structures/SkyraCommand';
+import { GuildSettings } from '@lib/types/settings/GuildSettings';
+import { BrandingColors } from '@utils/constants';
+import { PermissionLevels } from '@lib/types/Enums';
+
+export default class extends SkyraCommand {
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			cooldown: 10,
+			description: language => language.tget('COMMAND_SUGGEST_DESCRIPTION'),
+			extendedHelp: language => language.tget('COMMAND_SUGGEST_EXTENDED'),
+			requiredPermissions: ['EMBED_LINKS'],
+			runIn: ['text'],
+			usage: '<suggestion:string{3,2000}>'
+		});
+	}
+
+	public async run(message: KlasaMessage, [suggestion]: [string]) {
+		console.log('vibecheck passed');
+		const suggestionsChannelID = message.guild!.settings.get(GuildSettings.Suggestions.SuggestionsChannel) as string;
+
+
+		const suggestionsChannel = this.client.channels.get(suggestionsChannelID) as TextChannel;
+		// Get the next suggestion ID
+		const suggestionID = message.guild!.settings.get(GuildSettings.Suggestions.AscendingID);
+
+		// Post the suggestion
+		const suggestionsMessage = await suggestionsChannel.send(new MessageEmbed()
+			.setColor(BrandingColors.Primary)
+			.setAuthor(`${message.author.tag} (${message.author.id})`, message.author.displayAvatarURL({ size: 128 })!)
```

Comment:
Use the `getDisplayAvatar` util function to ensure png's are shown for non-animated avatars